src/attack.c

```diff
@@ -519,19 +521,32 @@ attack_host (struct scan_globals *globals, struct in6_addr *ip, GSList *vhosts,
                   kb_item_push_str (main_kb, "internal/results", buffer);
                   goto host_died;
                 }
+              else if (e == ERR_NO_FREE_SLOT)
+                {
+                  if (forks_retry < MAX_FORK_RETRIES)
+                    {
+                      forks_retry++;
+                      g_warning ("Launch failed for %s. No free slot available "
```

Comment:
```suggestion
                      g_warning ("Launch failed for %s. No free slot in the internal process table available "
```
Makes it more explicit what is meant with `slot`.